modules/connectivity/ConnectionQuality.js

```diff
@@ -278,6 +278,11 @@ export default class ConnectionQuality {
                     this._maybeUpdateUnmuteTime();
                 }
             });
+        conference.on(
+            ConferenceEvents.TRACK_MAX_ENABLED_RESOLUTION_CHANGED,
+            track => {
+                this._localStats.maxEnabledResolution = track.maxEnabledResolution;
```

Comment:
is the extra conference event really needed? I suppose `ConnectionQuality` could subscribe to the `RTC` instance directly?